test_scripts/RC/OnRCStatus/003_registration_not_rc_app.lua

```diff
@@ -5,7 +5,11 @@
 -- Requirement summary:
 -- [SDL_RC] TBD
 --
--- Description: SDL shall not send OnRCStatus notifications to not rc registered app
+-- Description:
+-- In case:
+-- 1) Not RC application is registered
```

Comment:
Not RC application is registered ==> No RC application is registered

Reply:
@yang1070 updated to non-Rc in 4f94bb15b488535907390542330636d3e45e5b58 , `no RC application is registered` is meaning not the same that i mean.